### PR TITLE
Documentation: add readme file for quote server

### PR DESCRIPTION
**Pull request by hairongchen**
Signed-off-by: Hairong Chen hairong.chen@intel.com

### CODE REVIEW

File: service/quote-server/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Service: CCNP Quote Service
+
+This service will provide quote generated by underlying TEE platform for remote attestation service to verify the integrity and confidentiality of the trusted computing environment and required software environment.
+
+## Introduction
+
+This service provides functionality to fetch quote of underlying TEE platform with nonce as mandatory input and a base64 encoded user data as optional input. The nonce and optional user data will be digested and added into quote for remote attestation to verify the freshness of the quote and the user specified data.
+
+The quote server use Unix domain socket based gRPC to serve the client SDK and the proto buffer message is as bellow:
```
Comment (hwang37) [inline service/quote-server/README.md:9]: 1. It's wired to use "Unix". Change to other word?  Linux? Network? ...
2. Separate 2nd sentence.
The proto buffer message definition is as below:

File: service/quote-server/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+message GetQuoteResponse {
+    string quote = 1;
+    string quote_type = 2;
+}
+
+```
+
+## Installation
+The quote service can be deployed as either DaemonSet or sidecar in different user scenarios.
```
Comment (hwang37) [inline service/quote-server/README.md:43]: in different user scenarios =>
according to different user scenario.

File: service/quote-server/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+### Build docker image
+The Dockerfile for the service can be found under `container/quote-server` directory. Use the following command to build the image:
+
+```
+docker build -t ccnp_get_quote:0.1 -f container/quote-server/Dockerfile .
+```
+
+> Note: if you are using containerd as the default runtime for kubernetes. Please remember to use the following command to import the image into containerd:
```
Comment (hwang37) [inline service/quote-server/README.md:52]: if you are using containerd as the default runtime for kubernetes, don't forget to use the following commands to import the image into containerd first:

File: service/quote-server/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+```
+docker save -o ccnp-quote-server.tar ccnp_get_quote:0.1
+ctr -n=k8s.io image import ccnp-quote-server.tar
+```
+
+### Deploy as DaemonSet in Kubernetes
+please edit file `service/quote-server/deployment/quote_server-daemonset.yaml` with the container image to use and run:
```
Comment (hwang37) [inline service/quote-server/README.md:59]: with the container image to use and run:
=>
with???

File: service/quote-server/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+```
+kubectl apply -f service/quote-server/deployment/quote_server-daemonset.yaml
+```
+
+## Testing
+One can play with service on host from the source code following the steps below:
```
Comment (hwang37) [inline service/quote-server/README.md:65]: User can play with quote service on host by following the steps below:

File: service/quote-server/README.md (hunk 1 above)
Comment (hairongchen) [reply service/quote-server/README.md:9]: UDS is a special term: https://en.wikipedia.org/wiki/Unix_domain_socket

File: service/quote-server/README.md (hunk 2 above)
Comment (hairongchen) [reply service/quote-server/README.md]: updated, thanks!

File: service/quote-server/README.md (hunk 4 above)
Comment (hairongchen) [reply service/quote-server/README.md:59]: updated, thanks!

File: service/quote-server/README.md (hunk 5 above)
Comment (hairongchen) [reply service/quote-server/README.md:65]: updated, thanks!

**Review by kenplusplus — APPROVED** [review]
LGTM

File: service/quote-server/README.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+> Note: if you are using containerd as the default runtime for kubernetes, don't forget to use the following commands to import the image into containerd first:
+```
+docker save -o ccnp-quote-server.tar ccnp_get_quote:0.1
+ctr -n=k8s.io image import ccnp-quote-server.tar
+```
+
+### Deploy as DaemonSet in Kubernetes
+please check file `service/quote-server/deployment/quote_server-daemonset.yaml` to confirm the container image to use and run:
```
Comment (Ruoyu-y) [inline service/quote-server/README.md:59]: please change path to the latest one.

File: service/quote-server/README.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+```
+kubectl apply -f service/quote-server/deployment/quote_server-daemonset.yaml
```
Comment (Ruoyu-y) [inline service/quote-server/README.md:61]: same here

**Review by Ruoyu-y — CHANGES_REQUESTED** [review]

File: .github/cspell.json [hunk 8]
```diff
@@ -15,9 +15,11 @@
     "words": [
         "ccnp",
         "containerd",
-        "Eventlog",
+        "daemonset",
         "grpcurl",
         "proto",
+        "quoteserver"
+        "Eventlog",
```
Comment (Ruoyu-y) [inline .github/cspell.json:22]: change the sequence?

File: .github/cspell.json [hunk 9]
```diff
@@ -15,9 +15,11 @@
     "words": [
         "ccnp",
         "containerd",
-        "Eventlog",
+        "daemonset",
```
Comment (Ruoyu-y) [inline .github/cspell.json:18]: where is 'daemonset' used?

**Review by Ruoyu-y — APPROVED** [review]

**Pull request merged**